filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/web/router/SpringHealthRouter.java

```diff
@@ -31,6 +31,10 @@
 import java.util.Map;
 import java.util.Optional;
 
+/**
+ * Route for "/health[/|/liveliness|/readiness|/{component}]" to the Spring {@link HealthEndpoint}
+ * or {@link ApplicationAvailability} objects
+ */
```

Comment:
Constructor at line 63 requires javadoc
Specify a type for status at line 74
SonarQube reports that mapSpringToAkka method should be static as private methods that dont access instance data can be static